app/account/account.rb

```diff
@@ -0,0 +1,27 @@
+class Account
+  attr_reader :cards, :name, :login, :password
```

Comment:
Where do we use name reader? And move attr_reader to private scope if we don`t use it outside.